workflow/rocoto/workflow_xml.py

```diff
@@ -118,6 +118,12 @@ def _get_cycledefs_cycled(self):
         sdate = sdate + to_timedelta(interval)
         strings.append(f'\t<cycledef group="gdas">{sdate.strftime("%Y%m%d%H%M")} {edate.strftime("%Y%m%d%H%M")} {interval}</cycledef>')
 
+        if self._app_config.do_jedilandda:
+            sdate_land = sdate.replace(hour=18, minute=0, second=0)
+            edate_land = edate
+            if edate >= sdate:
+                strings.append(f'\t<cycledef group="gdas_land_prep">{sdate_land.strftime("%Y%m%d%H%M")} {edate_land.strftime("%Y%m%d%H%M")} 24</cycledef>')
```

Comment:
```suggestion
                strings.append(f'\t<cycledef group="gdas_land_prep">{sdate_land.strftime("%Y%m%d%H%M")} {edate_land.strftime("%Y%m%d%H%M")} 24:00:00</cycledef>')
```

Reply:
I think this is the problem: the interval in the cycledef doesn't have the zeros for minutes and seconds. Compare to the others in the XML. So there is a prep land obs job for almost every minute (I'm not sure how '24' translates into 'most minutes'.)